state/machine_upgradeseries.go

```diff
@@ -260,7 +260,7 @@ func (m *Machine) RemoveUpgradeSeriesLock() error {
 			return nil, errors.Trace(err)
 		}
 		if !locked {
-			return nil, errors.NotFoundf("upgrade series lock for machine %q", m)
+			return nil, jujutxn.ErrNoOperations
```

Comment:
Remove of lock is idempotent and as such does not throw an error when a lock is not found.